pkg/cmd/pulumi/org_search_ai.go

```diff
@@ -69,21 +78,29 @@ func (cmd *searchAICmd) Run(ctx context.Context, args []string) error {
 	if err != nil {
 		return err
 	}
-	var filterName string
 	if cmd.orgName == "" {
-		filterName = userName
-	} else {
-		filterName = cmd.orgName
+		cmd.orgName = userName
```

Comment:
This bit will be replaced when https://github.com/pulumi/pulumi/pull/13879 finishes merging, and I can rebase to the default org implementation.